src/providers/formattingProvider.ts

```diff
@@ -279,6 +279,24 @@ export const internalFormat = (
     const trimSpaces = options.trimExtraSpaces;
 
     // REGULAR EXPRESSION
+    /** Formatter's directive `;@AHK++AlignAssignmentOn` */
+    const ahkAlignAssignmentOn = /;\s*@AHK\+\+AlignAssignmentOn/i;
+    /** Formatter's directive `;@AHK++AlignAssignmentOff` */
+    const ahkAlignAssignmentOff = /;\s*@AHK\+\+AlignAssignmentOff/i;
+    /** Formatter's directive `;@AHK++FormatBlockCommentOn` */
+    const ahkFormatBlockCommentOn = /;\s*@AHK\+\+FormatBlockCommentOn/i;
+    /** Formatter's directive `;@AHK++FormatBlockCommentOff` */
+    const ahkFormatBlockCommentOff = /;\s*@AHK\+\+FormatBlockCommentOff/i;
+    /**
+     * Semicolon must have at least one space or tab to its left or semicolon
+     * placed at the beginning of a line.
+     *
+     * Example: `ToolTip;NotComment`, `ToolTip ;Comment`
+     *
+     * Must be in sync with comment regexp in `purify()` (formatting utility
```

Comment:
"Must be in sync" -- can we make them both an imported variable from some "constants.ts" file to enforce this?

Reply:
Done.